src/Sowp/NewsModuleBundle/Resources/views/news/index.html.twig

```diff
@@ -0,0 +1,136 @@
+                                </td>
+                            </tr>
+                            <tr>
+                                <th>
+                                    Created By
+                                </th>
+                            </tr>
+                            <tr>
+                                <td>
+                                    {{ news_entry.createdBy }}
+                                </td>
+                            </tr>
+                            {% if news_entry.modifiedAt %}
+                                <tr>
+                                    <th>
+                                        Last Modified At
+                                    </th>
+                                </tr>
+                                <tr>
+                                    <td>
+                                        {{ news_entry.modifiedAt|date('Y-m-d H:i:s') }}
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <th>
+                                        Last Modified By
+                                    </th>
+                                </tr>
+                                <tr>
+                                    <td>
+                                        {{ news_entry.modifiedAt|date('Y-m-d H:i:s') }}
+                                    </td>
+                                </tr>
+                                {% if news_entry.modifyNote is defined and news_entry.modifyNote|length %}
+                                    <tr>
+                                        <th>
+                                            Modify Note
+                                        </th>
+                                    </tr>
+                                    <tr>
+                                        <td>
+                                            {{ news_entry.modifyNote }}
+                                        </td>
+                                    </tr>
+                                {% endif %}
+                                {% if news_entry.deletedAt %}
+                                    <tr>
+                                        <th>
+                                            Deleted At
+                                        </th>
+                                    </tr>
+                                    <tr>
+                                        <td>
+                                            {{ news_entry.deletedAt|date('Y-m-d H:i:s') }}
+                                        </td>
+                                    </tr>
+                                {% endif %}
+                            {% endif %}
+                        </tbody>
+                    </table>
+                    <hr>
+                    <div class="container-liquid" style="word-wrap: break-word">
```

Comment:
Omawialiśmy to wcześniej.....
Był problem przy pisaniu jednej długiej linii w TinyMCE. 
Na chwilę tamtą stwierdzone zostało że zajmiemy się tym dokładniej przy nakładaniu frontów, a nie wiem czy pamiętasz powiedziałem że poprawie jakkolwiek ten tymczasowy html żeby poprostu trzymał strukturę... więc jest to tylko do czasu zmiany htmla